client/post-editor/editor-drawer/featured-image.jsx

```diff
@@ -32,25 +36,50 @@ class EditorDrawerFeaturedImage extends Component {
 		stats.recordEvent( 'Featured image removed' );
 	}
 
+	onFilesDrop = ( files ) => {
+		Dispatcher.register( function( payload ) {
```

Comment:
From bottom to top:
> I'm assuming it's a temporary thing, but right now this obviously will add a new handler to Dispatcher on each filesdrop event.

Yes, this is temporary, just to get the functionality working. 

> why is it needed?

> if it is, could this be better accomplished by relaying these into Redux?
`MediaActions` is currently sending events for the upload state through Flux. I want to bring that to Redux if possible.

> if it is, can we extend a preëxisting Dispatcher.register callback?

> if it is, could this be better accomplished with one-off subscriptions (e.g. eventEmitter.once( cb ))?

If we manage to push the state updates to Redux, these won't be needed at all.